src/tye/InitHost.cs

```diff
@@ -105,5 +110,30 @@ private static void ThrowIfTyeFilePresent(FileInfo? path, string yml)
                 throw new CommandException($"File '{tyeYaml}' already exists. Use --force to override the {yml} file if desired.");
             }
         }
+
+        private static void AddToSolutionItems(FileInfo? path)
+        {
+            var fileGuid = Guid.NewGuid().ToString("B").ToUpper();
+
+            var slnFiles = Directory.GetFiles(path!.DirectoryName, "*.sln");
+            // noop if there are no sln files
+            if (slnFiles.Length == 1)
+            {
+                // assume the first one is the one wanted.
+                var sln = slnFiles[0];
+                //format
+                var insertItem = @$"Project(""{fileGuid}\"") = ""Solution Items"", ""Solution Items"", ""{fileGuid}""
```

Comment:
A few things here:
1. Why is there a backslash `\` after fileGuid? Looks like that's not needed since you already have `""` which should resolve correctly to a quotation mark.
2. How did you come up with this format for insert Item? Specifically, from my understanding of [this doc](https://docs.microsoft.com/en-us/visualstudio/extensibility/internals/solution-dot-sln-file?view=vs-2019#file-body) the two Guids should not be the same, one is a project Guid, one is a project type Guid. The current logic creates a random Guid and uses it for both, which doesn't seem right. Have you tested this logic? What does the sln file look like after you open it?
3. This logic assumes it's safe to insert a folder called `Solution Items`, what if that folder already exists in the sln file?

Reply:
I tested it out and it seems like adding this section worked:
```
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{D85F58E9-CD8F-45D7-B2F2-ADAE5812E9C0}"
+       ProjectSection(SolutionItems) = preProject
+               tye.yaml = tye.yaml
+       EndProjectSection
+EndProject
```
Still needs logic to resolve whether `Solution Items` already exists in the sln file.

Reply:
Thanks @JunTaoLuo for the comments here. The `\` was a miss on the last edit, will fix this and I'll make a quick change to check for the existing "Solution Items".